<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Soham" failures="1" tests="4" name="Test" time="16.197" errors="0" timestamp="2025-02-06T17:03:06 IST">
  <testcase classname="assertion.HardAssertion" name="checkWindowTitle" time="3.09"/>
  <testcase classname="assertion.HardAssertion" name="checkUrl" time="4.019">
    <failure type="java.lang.AssertionError" message="expected [https://katalon-demo-cura.herokuapp.com/] but found [http://www.facebook.com]">
      <![CDATA[java.lang.AssertionError: expected [https://katalon-demo-cura.herokuapp.com/] but found [http://www.facebook.com]
at assertion.HardAssertion.checkUrl(HardAssertion.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkUrl -->
  <testcase classname="assertion.HardAssertion" name="checkHeadingDisplayed" time="3.492"/>
  <testcase classname="assertion.HardAssertion" name="checkMakeAppointemnt" time="0.001">
    <skipped/>
  </testcase> <!-- checkMakeAppointemnt -->
</testsuite> <!-- Test -->
